Python/Product/Analysis/ProjectEntry.cs

```diff
@@ -153,7 +167,7 @@ internal sealed class ProjectEntry : IPythonProjectEntry, IAggregateableProjectE
             }
 
             lock (this) {
-                _analysisVersion++;
+                AnalysisVersion++;
```

Comment:
This is the only place where `AnalysisVersion` is accessed under lock

Reply:
It's almost the only place where it's modified. I'll lock the other one, but most of the reads are fine.